app/models/user.rb

```diff
@@ -0,0 +1,32 @@
+require 'bcrypt'
+require 'dm-validations'
+
+class User
+  include DataMapper::Resource
+
+
+  property :id,                 Serial
+  property :name,               String, required: true
+  property :username,           String, unique: true, required: true, message: "That username is already taken"
```

Comment:
Metrics/LineLength: Line is too long. [111/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)